tools/st2_deploy.sh

```diff
@@ -521,9 +523,6 @@ else
   echo "  st2 is installed and ready to use."
 fi
 
-if [ ${INSTALL_WEBUI} == "1" ]; then
-  echo "  WebUI at http://`hostname`:9101/webui/"
-fi
 echo "=========================================="
```

Comment:
Because now it is the part of st2ctl. See https://github.com/StackStorm/st2/pull/1362/files#diff-0c83a67f0358e42d5815eef73dad98adR319